### DISCUSSION TITLE
[Support] I need example URLs of CommaFeed

### DISCUSSION THREAD

**Discussion by sjehuda** [Athou/commafeed#1315] [Q&A]
Greetings!

My name is Schimon and I have created a Userscript that parses web feeds by only using Javascript.

A friend has asked for [CommaFeed support](https://openuserjs.org/scripts/sjehuda/Newspaper/issues/Subscript_WCommafeed), which I would be glad to add.

I do not use CommaFeed, even though it is proudly mentoned in the documentation of my project.

Please kindly provide me with sample URLs of adding feeds to CommaFeed.

Thank you in advance!

**Comment by sjehuda** [comment]
@steve0greatness please help.

> Please kindly provide me with sample URLs of adding feeds to CommaFeed.

**Reply by Athou** [reply]
Hi @sjehuda,

If you're talking about the URL you can use to subscribe to a feed automatically in CommaFeed with a GET request, it's https://www.commafeed.com/rest/feed/subscribe?url=FEED_URL_HERE

**Reply by sjehuda** [reply]
Thank you very much!

@Steve0Greatness, I have added an option in settings dialog to add CommaFeed support.

More information at https://openuserjs.org/scripts/sjehuda/Newspaper/issues/Subscript_WCommafeed#comment-18e936ed69a

Please test and advise!

**Reply by sjehuda** [reply]
Good day @Steve0Greatness

Did you try the recent release of the userscript?

Can I close the ticket at OUJS?
